developer-docs-site/docs/guides/nfts/mint-nft-cli.md

```diff
@@ -4,587 +4,291 @@ title: "Mint NFTs with the Aptos CLI"
-
-See [create_nft_with_resource_account.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/2-Using-Resource-Account/sources/create_nft_with_resource_account.move) to better understand the related functions:
-
-* `init_module` - stores the resource account's signer capability within `ModuleData` for later usage.
-* `mint_event_ticket` - creates a resource signer by calling `account::create_signer_with_capability(&module_data.signer_cap)` to programmatically sign for `token::mint_token()` and `token::direct_transfer()` functions.
-
-### Mint an NFT to the nft-receiver account
-
-1. Run the following command to mint an NFT to the `nft-receiver` account, replacing `<resource-account-address>` with the relevant address from `.aptos/config.yaml`:
-
-```shell
-aptos move run --function-id <resource-account-address>::create_nft_with_resource_account::mint_event_ticket --profile nft-receiver
-```
-
-2. As before, receive output asking you to accept a gas fee for the transaction and enter: `yes`.
-
-3. Receive output containing:
-
-```shell
-"vm_status": "Executed successfully"
-```
-
-4. [See the module on Aptos Explorer](#see-the-module-on-aptos-explorer) as before by searching for its `transaction_hash`.
-
-## 3. Add admin account and functions
-
-Publishing a module under a resource account means that we will not be able to update the module, and the module will be immutable and autonomous. We need administrator accounts, because we give up control over the smart contracts when we deploy a module using a resource account. Admin accounts and functions let you update the configuration going forward.
-
-If you will need to update the configuration of the module, we suggest adding a few admin functions to control the configurations of the module. You will also need to add the admin account's address to the module's `Move.toml` file so the admin functions can confirm the caller to the admin functions is actually a valid administrator.
-
-This work maps to the demonstration in [reate_nft_with_resource_and_admin_accounts.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/3-Adding-Admin/sources/create_nft_with_resource_and_admin_accounts.move).
-
-### Create admin account
-
-Here we create an admin account and include two admin functions so we can set and update when/if we want to enable token minting for this contract.
-
-Run `aptos-init` once again, this time to create an admin account:
-
-```shell
-`aptos init --profile admin`
+```rust
+struct MintConfiguration has key {
+    collection_name: String,
+    token_name: String,
+    token_uri: String,
+}
 ```
 
-### Publish the module under a resource account
+We give `MintConfiguration` the `key` ability so that the module contract can store it at an account's resources, which the module can find later if supplied with the account address. The account address we store it to will be the creator of the collection.
```

Comment:
We should talk about how this MintConfiguration will be initially stored as a resource in the creator's account. So we can later say we'll move this to the resource account